movies-app/src/components/SortControl/SortControl.jsx

```diff
@@ -1,26 +1,57 @@
-import React from "react";
-import './SortControl.css';
+import React, { useState } from "react";
+import "./SortControl.css";
+import { useSearchParams, useLocation } from "react-router-dom";
 
-const SortControl = (props) => {
+const SortControl = () => {
+
+  const location = useLocation();
+  const [searchParams, setSearchParams] = useSearchParams({});
+  const [sortOption, setSortOption] = useState("release_date");
+  const activeGenre = location.search !== "" ? searchParams.get("activeGenre") : "All";
+  const searchQuery = location.search !== "" ? searchParams.get("search") : "";
```

Comment:
Így egyszerűbb lenne szerintem de ugyanúgy lehetne useStatetel mint felette
```suggestion
  const activeGenre =searchParams.get("activeGenre") || "All";
  const searchQuery = searchParams.get("search") || "";
```

Reply:
a useSate() logikat kiszedtem, mert reolad-nal az url parametereit kell hasznalnia